### PR TITLE
tech: Cleanup example config from prod elements

### PR DESCRIPTION
**Pull request by taratatach**
The example config should not contain elements such as URLs from our
production environment as people might simply copy the file without
changing some parts and end up making unnecessary requests to our
servers.

### CODE REVIEW

File: cozy.example.yaml [hunk 1]
```diff
@@ -345,21 +345,21 @@ registries:
 
 # Wizard used for moving a Cozy from one place/hoster to another
 move:
-  url: https://move.cozycloud.cc/
+  url: https://move.example.org/
```
Comment (nono) [inline cozy.example.yaml:348]: Well, in that case, `move.cozycloud.cc` is fine and should be kept

File: cozy.example.yaml [hunk 2]
```diff
@@ -457,7 +457,7 @@ contexts:
     default_redirection: drive/#/folder
     # This domain will be used as a suggestion for the members of a sharing
     # when they are asked for the URL of their Cozy instance
-    sharing_domain: mycozy.cloud
+    sharing_domain: example.org
```
Comment (nono) [inline cozy.example.yaml:460]: I think we should also keep this one

File: cozy.example.yaml (hunk 1 above)
Comment (taratatach) [reply cozy.example.yaml:348]: I was going to but the flagship config has a `move_url` attribute with a different URL (see https://github.com/cozy/cozy-stack/pull/4500/files#diff-accceeea297d6b4bddb571eaab50129c26e0485c18c259d7822b192f3bc3df65L494) so I figured the move wizard did not have to be https://move.cozycloud.cc.

Comment (nono) [reply cozy.example.yaml:348]: It can be overloaded by context, but it is not used

Comment (taratatach) [reply cozy.example.yaml:348]: OK.

**Review by nono — APPROVED** [review]

**Pull request merged**